src/usr/local/www/classes/Form/Group.class.php

```diff
@@ -148,7 +148,8 @@ public function __toString()
 		$label = new Form_Element('label', false, ['for' => $target]);
 		$label->addClass('col-sm-'.Form::LABEL_WIDTH, 'control-label');
 
-		$title = htmlspecialchars(gettext($this->_title));
+		if (!empty(trim($this->_title)) || is_numeric($this->_title))
+			$title = htmlspecialchars(gettext($this->_title));
```

Comment:
So not using {} for one liners seems to be the norm in all the classes code.  So was just following what seemed to be the norm there.  Given that.  Should the {} be added or not?  I'm okay either way.  Just let me know the official code style policy for the classes and I'll do that.  Thanks.